p3/plot/_navchart.py

```diff
@@ -9,10 +9,12 @@
 import pandas as pd
 
 from p3._utils import _require_columns, _require_numeric
+from p3.plot._common import Legend, ApplicationStyle
 from p3.plot._cascade import _get_colors
+from p3.plot.backend.matplotlib import NavChart
 
 
-def navchart(pp, cd, eff=None, goal=None, **kwargs):
+def navchart(pp, cd, eff=None, size=(5, 5), goal=None, **kwargs):
```

Comment:
The default values for `size` don't match what's in the docstring

Reply:
Good catch!  Copy-paste is dangerous. 😄 

Fixed in 42c16d099829a51c8c727e35e84d2f1d63db6f3c.